package-lock.json

```diff
@@ -1,1323 +1,8 @@
 {
   "name": "setup-kind",
-  "version": "0.5.0",
-  "lockfileVersion": 2,
+  "version": "0.6.0",
+  "lockfileVersion": 1,
```

Comment:
You might want to update your npm version if you want to move to version 2 of lock file, it seems.